about.html

```diff
@@ -0,0 +1,153 @@
+      <li><a href="">Program</a></li>
+      <li><a href="">Join</a></li>
+      <li><a href="">Sponsor</a></li>
+      <li><a href="">News</a></li>
+    </ul>
+  </div>
+  <header class="desktop-header">
+    <ul class="online-icons">
+      <li><a class="iconx fb" href="/">
+        <i class="fab fa-facebook-square"></i>
+        </a>
+      </li>
+      <li> <a class="iconx tw" href="/">
+        <i class="fab fa-twitter"></i>
+      </a></li>
+      <li class="iconx en"> <a class="iconx" href="">English</a></li>
+    </ul>
+  </header>
+
+  <div class="desktop-nav">
+    <img src="./images/firstnavlogo.png" alt="">
+    <ul class="dsk-nav-items">
+      <li class="nav1"><a class="nav1" target="_self" href="index.html">Home</a></li>
+      <li class="nav2">Program</li>
+      <li class="nav3" >Sponsor</li>
+      <li class="nav4" >News</li>
+    </ul>
+    <button class="last-logo">CC Campaign</button>
+  </div>
+
+  <div class="introduction">
+    <div class="burger" >
+      <a href="/"><i class="fas fa-bars hamburger-icon "></i></a>
+    </div>
+    <div class="heading">
+      <h4 class="heading1">"Hello! Sport lovers"</h4>
+
+      <h1 class="heading2">TotalEnergy AFCON Cameroon 2021</h1>
+    </div>
+    <div class="text11">
+      <p class="text1">A global come together for a joyful celebration and believing in the value of oneness, openness,
+        sharing and creating a positive change with participants from over 24 countries taking place in
+        Yaounde-Cameroon participants from over 24 countries taking place in
+        Yaounde-Cameroon .</p>
+    </div>
+    <div class="text22">
+      <p class="text2">Please contact us via email for any further questions concerning the AFCON</p>
+      <p class="text3">AFCON2021@Yaounde</p>
+    </div>
+  </div>
+  <div class="creation">
+    <div class="creation-text">
+      <p class="cc-text1"> TotalEnergy Afcon CAMEROON 2021 official logo</p>
+      <p class="cc-text2"> Afcon logo gotten through the delegation of CAF 2020</p>
+    </div>
+    <div class="creation-logo">
+      <span class="creation-img">
+        <img class="creation-img" src="./images/caflogo.png" alt="CAF">
+      </span>
+    </div>
+  </div>
+  <div class="the-past">
+    <div class="past-text">
+      <p class="past1">See the past AFCON ballot conferences</p>
+      <p class="past2">Take a look at the last two afcon conferences that took place in Egypt</p>
+    </div>
+    <div class="past-images">
+      <span class="past-img1">
+        <img src="./images/past1.png" alt="past-images">
+      </span>
+      <span class="past-img2">
+        <img src="./images/past2.png" alt="past-images">
+      </span>
+    </div>
+  </div>
+  <section class="partner">
+    <h2 class="partner-text">Partner</h2>
+    <ul class="partner-icons">
+      <li class="bet">1XBet</li>
+      <li class="visa">VISA</li>
+      <li class="orange">Orange-Money</li>
+      <li class="energies">Total-Energies</li>
+    </ul>
+  </section>
```

Comment:
- [ ] The _Partner_ section of your about page is overflowing on mobile screen. Kindly ensure to improve upon this aspect.

**Your mobile view (about page):**
![image](https://user-images.githubusercontent.com/12745474/133616824-fe23ea1a-6822-453e-8eba-f08f7eeeedee.png)

**The original mobile view (about page):**
![image](https://user-images.githubusercontent.com/12745474/133617534-65460d83-7379-46b3-a9b3-358be4376d7d.png)

Reply:
This issue is well resolved! ✔️

Reply:
[ ] OPTIONAL
Please update the **font family** of the title _Partner_